api/net/Dockerfile.aspnet

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
New image that we build locally and push to our image registry manually.  This is to hopefully reduce the size of the overall build.  We need to find a way to reduce the size because Buildah is limited to 10Gb and we're out of drive space...